README.md

```diff
@@ -114,8 +114,25 @@ directory the variables from `.env` will be loaded into the environment.
 
 E.g. `foreman run rails server` or `foreman run rails console`.
 
+### Google Places API (optional)
+
+To avoid accidentally exceeding the rate limit on [Google's Places API][google-places] (e.g. when heavily testing city-autocomplete fields) and thus blocking its usage for other RGSoC sites and devs:
+
+1. [Get your own key][google-places]
+
+1. Copy the `.env-example` file to `.env` and replace `YourGoogleMapsAPIKey` with your key
+
+1. Run the `foreman` command before any command
+  ```sh
+  # e.g.
+  foreman run rails server
```

Comment:
I wasn't aware that foreman honours `.env`. For some reason I always thought we were using [dotenv](https://github.com/bkeepers/dotenv) 👍